.github/workflows/check-pr-diagnostics-plugin.yaml

```diff
@@ -81,10 +81,15 @@ jobs:
       - build-release
     runs-on: ${{ needs.setup-runner.outputs.ec2-instance-id }} # run the job on the newly created runner
     steps:
+      - name: Set up Go 1.x
```

Comment:
This should be removed. This is already done above in the build-release section.

Reply:
strange but I think for different jobs we have to do different setup i removed that thing and workflow failed https://github.com/vmware-tanzu/community-edition/runs/6205983295?check_suite_focus=true#step:2:9

Reply:
upto my knowledge jobs run on separate machine 
[found this in workflow doc](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions#:~:text=Each%20job%20will%20run%20inside%20its%20own%20virtual%20machine%20runner%2C%20or%20inside%20a%20container%2C%20and%20has%20one%20or%20more%20steps%20that%20either%20run%20a%20script%20that%20you%20define%20or%20run%20an%20action%2C%20which%20is%20a%20reusable%20extension%20that%20can%20simplify%20your%20workflow.)

Reply:
We should make this one job with multiple steps then. I don't see a strong reason to have two separate jobs if it adds redundancy. But that's something we can follow up on later. Thanks!